content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -19,66 +19,68 @@ The features described on this page are in beta. Contact your Customer Success M
 - **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database
 - **Architecture**: Multi-tenant
 
-## Prerequisites
+## Overview
 
-### Install the Oracle integration
-
-On the [Integrations][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an [Oracle dashboard][5] in your account which can be used to monitor the performance of your Oracle databases.
-
-### Upgrade the Oracle integration in your Agent
-
-You can skip this step if this is your first time installing the Oracle integration. If you already installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
-
-Use the following command to migrate the Oracle integration from the legacy integration to the new one:
-
-```shell
-cp /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
-```
+### Prerequisites
 
-Deactivate the legacy integration:
+To configure Database Monitoring for Oracle, the following prerequisites must be met:
 
-```shell
-mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle.d/conf.yaml.bak
-```
+1. An [Agent version](#recommended-agent-version) that supports Oracle monitoring features must be installed.
+    - [Install the Agent](#install-the-agent)
+    - [Upgrade an existing Agent installation](#upgrade-an-existing-agent-installation)
+2. The Oracle integration must be installed.
+    - [Install the Oracle integration](#install-the-oracle-integration)
+    - [Verify an existing Oracle integration installation](#verify-your-existing-oracle-integration)
 
-Deactivating the legacy integration prevents sending the system metrics twice.
+### Setup
+If the above prerequisites are met, follow the setup instructions for your hosting type:
+{{< partial name="dbm/dbm-setup-oracle" >}}
```

Comment:
This is probably scope creep for this PR and could be an iterative update, but it seems like we could get more specific images for this partial. It's repetitive to the point where there's not really a point to having it in a partial. 
<img width="1165" alt="image" src="https://github.com/DataDog/documentation/assets/28788585/0f0d5b7c-96ec-4ab8-ae2f-e67e77f89618">

Reply:
I added a ticket to my queue, thank you!